shopfloor_single_product_transfer_mobile/static/src/scenario/single_product_transfer.js

```diff
@@ -0,0 +1,197 @@
+                />
+                <item-detail-card
+                    :key="make_state_component_key(['product', state.data.move_line.product.id])"
+                    :options="product_detail_options_for_set_quantity()"
+                    :record="state.data.move_line"
+                    :card_color="utils.colors.color_for('screen_step_done')"
+                />
+                <item-detail-card
+                    :key="make_state_component_key(['location_dest', state.data.move_line.location_dest.id])"
+                    :record="state.data.move_line.location_dest"
+                    :card_color="utils.colors.color_for('screen_step_todo')"
+                />
+                <v-card class="pa-2" :color="utils.colors.color_for('screen_step_todo')">
+                    <packaging-qty-picker
+                        :key="make_state_component_key(['packaging-qty-picker', state.data.move_line.id])"
+                        v-bind="utils.wms.move_line_qty_picker_props(state.data.move_line)"
+                    />
+                </v-card>
+            </template>
+            <div v-if="state_in(['select_product', 'set_quantity'])" class="button-list button-vertical-list full">
+                <v-row align="center">
+                    <v-col class="text-center" cols="12">
+                        <cancel-button v-on:cancel="on_cancel"></cancel-button>
+                    </v-col>
+                </v-row>
+            </div>
+            <last-operation v-if="state_is('show_completion_info')" v-on:confirm="state.on_confirm"></last-operation>
+        </Screen>
+    `,
+    methods: {
+        screen_title: function () {
+            if (_.isEmpty(this.current_doc()) || this.state_is("select_document")) {
+                return this.menu_item().name;
+            }
+            const title = this.current_doc().record.name;
+            return title;
+        },
+        current_doc: function () {
+            const data = this.state_get_data("set_quantity");
+            if (_.isEmpty(data)) {
+                return null;
+            }
+            return {
+                record: data.move_line.product,
+                identifier: data.move_line.product.name,
+            };
+        },
+        product_detail_options_for_set_quantity: function () {
+            return {
+                title_action_field: {action_val_path: "product.barcode"},
+                key_title: "product.display_name",
+                fields: [{path: "lot.name", label: "Lot", action_val_path: "lot.name"}],
+            };
+        },
+        display_location_operations_progress: function () {
+            const progress_info = _.result(
+                this.state,
+                "data.location.progress_info",
+                false
+            );
+            if (progress_info) {
+                return (
+                    progress_info.operations_done && progress_info.operations_to_do > 0
```

Comment:
We want to display when there is something to do. By design, the done quantity should always be 0 as moves are posted when the line is processed
```suggestion
                    progress_info.operations_to_do > 0
```